main.go

```diff
@@ -215,11 +232,18 @@ func main() {
 
 			socketHandler := wsHandler().ServeHTTP
 
-			extraOriginStr := "https://create.arduino.cc, http://create.arduino.cc, https://create-dev.arduino.cc, http://create-dev.arduino.cc, http://create-staging.arduino.cc, https://create-staging.arduino.cc"
-
-			for i := 8990; i < 9001; i++ {
-				extraOriginStr = extraOriginStr + ", http://localhost:" + strconv.Itoa(i) + ", https://localhost:" + strconv.Itoa(i)
+			portPlain, err := getBindPort(8990, 9001)
+			if err != nil {
+				panic(err)
 			}
+			portSSL, err := getBindPort(portPlain+1, 9001)
```

Comment:
You can maybe add a comment here to explain why you're passing portPlain+1. I heard people confused by it in the office. But of course if you find a free port, the previous are all occupied, and there's no reason to cycle through them again.